app/models/filter_option.rb

```diff
@@ -0,0 +1,11 @@
+class FilterOption
+  attr_reader :value
+
+  def initialize(value)
+    @value = value
+  end
+
+  def underscore
+    value.gsub(/[^a-zA-Z. ]/, ' ').strip.split(' ').map(&:downcase).join('_')
+  end
+end
```

Comment:
Sorry, I didn't understand the purpose of this class. The PR description says "to play nicely with Rails form helpers" and the commit message does not tell much. Can you clarify, please?

Reply:
I can amend the commit message.

Reply:
The collection_check_boxes and collection_radio_buttons act on a collection of objects. It uses methods from the object instance for setting the `value`, `name` and `id` attributes and the content for each checkbox.